### PR TITLE
pic18 device 18 get

### PR DESCRIPTION
**Pull request by Nitish1802**
PIC18 series device ID :
New AT Command added to get the device ID of the controller used .
Command = AT+DEVID?

### CODE REVIEW

**Review by sramrajkar — COMMENTED** [review]
Looks good. Just some changes + additions

File: app/app_src/application.c [hunk 1]
```diff
@@ -457,6 +457,58 @@ static void cmdI()
...
+        devId = ((uint16_t)devId1 << 8u) | devId2;
+        
+    #elif (__AVR128DA32__)
+        devId1 = SIGROW.DEVICEID0;
+        devId2 = SIGROW.DEVICEID1;
+        devId3 = SIGROW.DEVICEID2;
+        
+        devId_avr = ((devId1 << 16u) | (devId2 << 8u) | (devId3));
+        printf("DEVICE ID:%X,AVR128DA32\r\n", devId);
+    #endif
```
Comment (sramrajkar) [inline app/app_src/application.c:496]: Missing PIC32 device ID read instructions. Currently code just prints uninitialized variable when built for PIC32

**Review by sramrajkar — APPROVED** [review]

**Comment by sramrajkar** [comment]
You may have to rebase

**Comment by Nitish1802** [comment]
Rebased and cherry picked in #110 . Closing PR

**Pull request closed without merging**